lib/modules/manager/github-actions/extract.spec.ts

```diff
@@ -76,6 +100,20 @@ describe('modules/manager/github-actions/extract', () => {
           replaceString: '"actions/checkout@v1.1.2"',
           versioning: 'docker',
         },
+        {
+          currentValue: 'latest',
```

Comment:
I think we should add a skip reason, because we can't update latest values. it would print a datasource warning on lockup.